charts/warpstream-agent/templates/_helpers.tpl

```diff
@@ -118,3 +118,18 @@ Convert units like 4GiB and 2000MiB to bytes.
 {{- end -}}
 {{- $bytes -}}
 {{- end -}}
+{{/*
+
+Convert CPU requests to millicores integers like 1/2000m to 1000/2000.
```

Comment:
This comment confused me because i thought you were saying it converts 1/2000m to 1000/2000 but actually what it does is convert 1/2000m to 1/2000 and 1 to 1000/1000

Reply:
let me correct it